.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,10 @@
 ## Description
-Include a few sentences describing the overall goals for this PR (pull request). If applicable also add screenshots.
+<!--Include a few sentences describing the overall goals for this PR (pull request). If applicable also add screenshots.-->
```

Comment:
Actually, I'd prefer to make the below check list shorter instead of hiding this line.
It's IMO the most important one (although quite often skipped), whereas things below like "this PR passes all existing unit tests" are pretty pointless.

Reply:
Oh wait, this will only be hidden once the PR is opened but still shown when entering data? If it works like that, that's a good thing (but doesn't change the point of the superfluous checkboxes).

Reply:
Yes, it will only hide it in the preview. I think I will use `Include **below** a few sentences describing` to avoid people replacing the text between the tags (hence display nothing).
Any checkbox other than the units tests row, you'd like to remove?